base/applications/games/winmine/lang/ar-AR.rc

```diff
@@ -0,0 +1,83 @@
+/*
+ * PROJECT:    WineMine
+ * LICENSE:    GPL - See COPYING in the top level directory
+ * FILE:       base/applications/calc/lang/ar-AR.rc
```

Comment:
We do not mention the filename within headers anymore. That style is deprecated.